examples/anelastic_bomex.jl

```diff
@@ -122,18 +116,22 @@ set!(ρvᵍ, ρᵣ * ρvᵍ)
 
 @inline function Fρu_geostrophic(i, j, k, grid, clock, fields, parameters)
     f = parameters.f
+    v_avg = parameters.v_avg[i, j, k]
```

Comment:
probably the right way to derive this is to compute the average momentum directly, eg

```julia
ρv_avg = Field(Average(model.momentum.ρv, dims=(1, 2)))
```